public/app/features/admin/Users/UsersTable.tsx

```diff
@@ -0,0 +1,177 @@
+          ]
+        : []),
+      {
+        id: 'lastSeenAtAge',
+        header: 'Last active',
+        headerTooltip: {
+          content: 'Time since user was seen using Grafana',
+          iconName: 'question-circle',
+        },
+        cell: LastSeenAtCell,
+        sortType: (a, b) => new Date(a.original.lastSeenAt!).getTime() - new Date(b.original.lastSeenAt!).getTime(),
+      },
+      {
+        id: 'authLabels',
+        header: 'Origin',
+        cell: ({ cell: { value } }: Cell<'authLabels'>) => (
+          <>{Array.isArray(value) && value.length > 0 && <TagBadge label={value[0]} removeIcon={false} count={0} />}</>
+        ),
+      },
+      {
+        id: 'isDisabled',
+        header: '',
+        cell: ({ cell: { value } }: Cell<'isDisabled'>) => <>{value && <Tag colorIndex={9} name={'Disabled'} />}</>,
+      },
+      {
+        id: 'edit',
+        header: '',
+        cell: ({ row: { original } }: Cell) => {
+          return (
+            <a href={`admin/users/edit/${original.id}`} aria-label={`Edit team ${original.name}`}>
+              <Tooltip content={'Edit user'}>
+                <Icon name={'pen'} />
+              </Tooltip>
+            </a>
+          );
+        },
+      },
+    ],
+    [showLicensedRole]
+  );
+  return (
+    <VerticalGroup spacing={'md'}>
+      <InteractiveTable columns={columns} data={users} getRowId={(user) => String(user.id)} />
+      {showPaging && (
+        <HorizontalGroup justify={'flex-end'}>
+          <Pagination numberOfPages={totalPages} currentPage={currentPage} onNavigate={onChangePage} />
+        </HorizontalGroup>
+      )}
+    </VerticalGroup>
+  );
+};
+
+const OrgUnitsCell = ({ cell: { value, row } }: Cell<'orgs'>) => {
+  const styles = useStyles2(getStyles);
+  return (
+    <div className={styles.row}>
+      <OrgUnits units={value} icon={'building'} />
+      {row.original.isAdmin && (
+        <Tooltip placement="top" content="Grafana Admin">
+          <Icon name="shield" />
+        </Tooltip>
+      )}
+    </div>
+  );
+};
+
+const LicensedRoleCell = ({ cell: { value } }: Cell<'licensedRole'>) => {
+  const styles = useStyles2(getStyles);
+
+  return (
+    <>
+      {value === 'None' ? (
+        <span className={styles.disabled}>
+          Not assigned{' '}
+          <Tooltip placement="top" content="A licensed role will be assigned when this user signs in">
+            <Icon name="question-circle" />
+          </Tooltip>
+        </span>
+      ) : (
+        value
+      )}
+    </>
+  );
+};
+
+const LastSeenAtCell = ({ cell: { value } }: Cell<'lastSeenAtAge'>) => {
+  const styles = useStyles2(getStyles);
+
+  return <>{value && <>{value === '10 years' ? <span className={styles.disabled}>Never</span> : value}</>}</>;
+};
+
+const getStyles = (theme: GrafanaTheme2) => {
+  return {
+    disabled: css`
+      color: ${theme.colors.text.disabled};
+    `,
+    row: css`
+      display: flex;
+      align-items: center;
+    `,
+  };
```

Comment:
prefer css objects rather than tagged template strings.

all of grafana UI uses css objects now, but core grafana is a lot to migrate...

Reply:
Will leave this for the follow up PRs, this one is already quite large.